pkg/predict/predictor.go

```diff
@@ -131,18 +132,45 @@ func (p *Predictor) Predict(inputs Inputs) (*Response, error) {
 	}
 	defer resp.Body.Close()
 
-	// TODO
-	if resp.StatusCode == http.StatusBadRequest {
+	if resp.StatusCode == http.StatusUnprocessableEntity {
 		body := struct {
-			Message string `json:"message"`
+			Detail []struct {
+				Location []string `json:"loc"`
+				Message  string   `json:"msg"`
+				Type     string   `json:"type"`
+			} `json:"detail"`
 		}{}
 		if err := json.NewDecoder(resp.Body).Decode(&body); err != nil {
-			return nil, fmt.Errorf("/predict call return status 400, and the response body failed to decode: %w", err)
+			return nil, fmt.Errorf("/predictions call returned status 422, and the response body failed to decode: %w", err)
 		}
-		if body.Message == "" {
-			return nil, fmt.Errorf("Bad request")
+
+		errorMessages := []string{}
+
+		for _, validationError := range body.Detail {
+			if len(validationError.Location) != 3 || validationError.Location[0] != "body" || validationError.Location[1] != "input" {
+				responseBody, _ := json.MarshalIndent(body, "", "\t")
+				return nil, fmt.Errorf("/predictions call returned status 422, and there was an unexpected message in response:\n\n%s", responseBody)
+			}
+
+			errorMessages = append(errorMessages, fmt.Sprintf("- %s: %s", validationError.Location[2], validationError.Message))
 		}
-		return nil, fmt.Errorf("Bad request: %s", body.Message)
+
+		validationErrorMessage := fmt.Sprintf(
+			`The inputs you passed to cog predict could not be validated:
+
+%s
+
+You can provide an input with -i. For example:
+
+    cog predict -i blur=3.5
+
+If your input is a local file, you need to prefix the path with @ to tell Cog to read the file contents. For example:
+
+    cog predict -i path=@image.jpg`,
+			strings.Join(errorMessages, "\n"),
+		)
+
+		return nil, fmt.Errorf(validationErrorMessage)
```

Comment:
`fmt.Errorf()` does formatting like `fmt.Sprintf()`, so this could be simplified a little by doing the formatting in `fmt.Errorf()`.